pages/user/generate.tsx

```diff
@@ -46,12 +46,14 @@ function GenerateUser() {
   return <Box component="form" 
       sx={{
         '& .MuiTextField-root': { m: 1, width: '100%' },
-        '& button': { m: 1 }
+        '& button': { m: 1 },
       }}
     >
+      <Box sx={{ m:1, width: "100%"}}>
       { message ? 
```

Comment:
This is inside the Box component so it needs to be indented one tab to the right